test/crawler_web/controllers/home_controller_test.exs

```diff
@@ -23,4 +26,72 @@ defmodule CrawlerWeb.HomeControllerTest do
       assert html_response(conn, 200) =~ "#{gettext("Log in")}</a>"
     end
   end
+
+  describe "POST /home" do
+    test "given valid keyword csv file, creates keyword successfully", %{conn: conn} do
+      user = insert(:user)
+      uploaded_file = keyword_file_fixture("valid.csv")
+
+      conn =
+        conn
+        |> log_in_user(user)
+        |> post(Routes.home_path(conn, :create), %{keyword_csv_file: %{file: uploaded_file}})
+
+      assert get_flash(conn, :info) == gettext("Keywords were uploaded!")
```

Comment:
should we assert the `created keyword from the database` here to make sure this controller creating the keywords in database

Reply:
Fix in [e30ade7](https://github.com/vnntsu/elixir-ic/pull/36/commits/e30ade7853ad15da4e2ca9ca7f92f5a63f2ee6df)